WhatsApp/app/src/main/java/com/example/whatsapp/ui/fragments/contactsselect/ContactsSelectAdapter.kt

```diff
@@ -0,0 +1,180 @@
+package com.example.whatsapp.ui.fragments.contactsselect
+
+import android.media.Image
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.MotionEvent
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import androidx.recyclerview.selection.ItemDetailsLookup
+import androidx.recyclerview.selection.ItemKeyProvider
+import androidx.recyclerview.selection.SelectionTracker
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import com.example.firestorerepository.datatypes.Contact
+import com.example.whatsapp.R
+import com.example.whatsapp.databinding.ItemContactBinding
+
+private const val LOGTAG = "ContactsSelectAdapterTest"
```

Comment:
This log string is too big, I am getting a warning it shold be max 23 characters